README.md

```diff
@@ -27,9 +27,13 @@ The application supports loading Guru cards from .json files.
 To load a JSON file containing Guru cards in your local environment, from within `/app`:
 
 ```bash
-make ingest-guru-cards DATASET_ID=some_identifier FILEPATH=path/to/some_cards.json
+make ingest-guru-cards DATASET_ID=dataset_identifier BENEFIT_PROGRAM=SNAP BENEFIT_REGION=Michigan FILEPATH=path/to/some_cards.json
 ```
 
+- The same `DATASET_ID` identifier can be used for multiple documents (`FILEPATH`) to represent that the documents belong in the same dataset.
+- Example `BENEFIT_PROGRAM` values include `housing`, `utilities`, `food`, `medical`, `employment`, `SNAP`, `Medicaid`.
```

Comment:
```suggestion
- Example `BENEFIT_PROGRAM` values include `housing`, `utilities`, `food`, `medical`, `employment`, `SNAP`, `Medicaid`, `multi_benefit`, `etc`.
```

Reply:
Adding this so it doesn't appear to be a restrictive list.

Reply:
I figured "Example" and "include" would convey that it's not a complete list.